.github/workflows/bump.yml

```diff
@@ -12,7 +12,7 @@ on:
 
 jobs:
   bump:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
The 20.04 runners were retired, but I don't think there's anything about these pipelines that would require a specific version of ubuntu. Since `release.yml` was already on this pattern, that's what I went with.

The alternative would be 24.04, but I'm not sure us bumping it manually every couple years buys anything.